tests/test_sequencer.py

```diff
@@ -157,16 +157,15 @@ def plan():
 def test_sequence_get_put():
     seq = SimSequencer('ECS:TST:100', name='seq')
 
-    dummy_sequence = [[i for i in range(0, 20)],
-                      [i for i in range(20, 40)],
-                      [i for i in range(40, 60)],
-                      [i for i in range(60, 80)]]
+    dummy_sequence = [[1,  2,  3,  4],
+                      [5,  6,  7,  8],
+                      [9, 10, 11, 12],
+                      [13, 14, 15, 16]]
```

Comment:
You might consider adding tests for scenarios where the user mistakenly makes a list such as:
`[1, 2, 3, 4], [5, 6, 78], ...`